docs/operations/installing_operator_into_cluster_en.md

```diff
@@ -13,7 +13,7 @@ The installation from GitHub requires the installation YAML which contains all r
 
 ```
 GITHUB_VERSION=0.0.6
-kubectl apply -f https://github.com/cloudogu/k8s-dogu-operator/v2/releases/download/v${GITHUB_VERSION}/k8s-dogu-operator_${GITHUB_VERSION}.yaml
```

Comment:
This documentation is very old and should be changed according the helm installation method.